gapis/api/gles/tweaker.go

```diff
@@ -91,6 +91,14 @@ func (t *tweaker) glDisable(ctx context.Context, name GLenum) {
 	}
 }
 
+func (t *tweaker) glViewport(ctx context.Context, x, y GLint, w, h GLsizei) {
+	if o := t.c.Rasterization().Viewport(); o.X() != x || o.Y() != y || o.Width() != w || o.Height() != h {
```

Comment:
Nit: You could use [`Rect.EqualTo`](https://github.com/google/gapid/blob/master/gapis/api/gles/math.go#L18)